changelog/2022-03-09.md

```diff
@@ -0,0 +1,30 @@
+
+## :wrench: March 9, 2022
+
+**CLI version: 0.8.4**
+
+:heavy_check_mark: Resolves an issue where using a relative path for the `dependency_file_info` property in a Run config was breaking. For example, this now works: 
+    ```# Dependency file specification
+        dependency_file_info:
+        package_manager: conda
+        path: ./env/env-deepcdl-pytorch.yml ```
+
+:heavy_check_mark: Support for specifying version of Julia image to use in Runs. We will support every patch release of julia from 1.6.1 up.
+
+`grid run --framework julia` will use the latest Julia version available (currently 1.7.1)
+
+`grid run --framework julia:X.Y.Z` will use Julia with the version X.Y.Z 
+
+<note>At this time, this feature is available for CLI users only</note>
+
+:heavy_check_mark: Runs will fail more quickly if there is an issue with image building.
+
+:heavy_check_mark: Resolves issue with `--num_trials` parameter being ignored.
+
+:heavy_check_mark: Logging improvements to silence noisy stacktraces.
+
+:heavy_check_mark: 'pytorch' and 'torch' are now both equal and acceptable inputs to the framework option for `grid run` 
+
+ex: `--framework pytorch` == `--framework torch`
```

Comment:
didn't we already announce this stuff?

Reply:
i did not - looks like these were all merges made since last Tuesday, and I haven't made any announcements since then

Reply:
there was a bug where "pytorch" was not working as an option and that was fixed and merged this week